client/app.jsx

```diff
@@ -215,14 +217,15 @@ class App extends React.Component {
   }
 
   componentDidMount() {
+    console.log(window.location.pathname.slice());
```

Comment:
active console log

Reply:
Ooh I've definitely got a couple more of those. I'll sniff em out and delete them! Thanks Andrew!